build.sh

```diff
@@ -21,29 +21,27 @@ fi
 npm ci --legacy-peer-deps
 rm -Rf build
 
-if [[ 1 == "$FORCE" ]]; then
-  rm -Rf dist
-else
+if [[ 1 != "$FORCE" ]]; then
   for item in `ls dist | grep -v cht-core`; do
     rm -rf dist/"$item"
   done
 fi
```

Comment:
Isn't that the same as `rm -Rf dist`?

Reply:
That loop seems basically equivalent to `rm -rF dist`. No?

I don't think we want to clear dist in case of this scenario:

> When we add support to the test-harness for a new version of cht-core and that requires a breaking change to our build process (such as what we have now), we stop building the broken CHT versions (remove them from cht_versions), but keep their dist/cht-core-* artifacts.

I'm thinking:

1. Remove --force mode from the build script
2. Run `rm -rF dist/$version` to ensure clean build of all supported versions in build script
3. When removing a version from build script, the author has the choice to remove the artifacts from dist (major version release) or leave them (minor version release)

Reply:
True True